lib/grpperm.gd

```diff
@@ -529,6 +529,30 @@ DeclareGlobalFunction( "ReducedPermdegree" );
 
 DeclareGlobalFunction("MovedPointsPerms");
 
+
+#############################################################################
+##
+#A  OrbitsMovedPoints( <G> )
+##
+##  <#GAPDoc Label="AllBlocks">
+##  <ManSection>
+##  <Attr Name="AllBlocks" Arg='G'/>
```

Comment:
I'm guessing this reference to `AllBlocks` (and two lines above) is a cut+paste error?